lib/run-executable.js

```diff
@@ -21,17 +21,20 @@ function runExecutable(args, cwd, logLevel) {
 
     const childProcess = spawn(pathToExecutable, args, {
       cwd,
-      stdio: ['inherit', 'inherit', 'pipe']
+      stdio: pipe
     });
 
     childProcess.on('error', (err) => {
       errors += err;
     });
```

Comment:
This is leftover broken code. Removing this would be ideal, since we're now piping output to the main process.

```suggestion
```